### PR TITLE
Fixed breadcrumb checks for Edit pages and few other improvements

### LINKED ISSUE #246: Typo in is_diplayed causing am_i_here() always return False
One of common checks inside view's is_dispayed property looks like this:
```python
self.breadcrumb.read().starts_with('Edit ')
```
But str's method is called `startswith`, with no underscore.
This causes exception, and corresponding am_i_here() always returns False.

### PR DESCRIPTION
**Pull request by abalakh**
* Fixed breadcrumb checks for Edit pages (fixes #246)
* Added manifest refresh, improvements for organization delete and subscription waiters

### CODE REVIEW

File: airgun/widgets.py [hunk 1]
```diff
@@ -691,6 +692,7 @@ def select_loc(self, loc_name):
         )
         self.browser.plugin.ensure_page_safe()
 
+    @property
```
Comment (oshtaier) [inline airgun/widgets.py]: should not you change https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/location.py#L87 too?

File: airgun/widgets.py [hunk 2]
```diff
@@ -676,6 +676,7 @@ def select_org(self, org_name):
...
```
Comment (oshtaier) [inline airgun/widgets.py]: should not you change https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/organization.py#L87 too?

File: airgun/entities/organization.py [hunk 3]
```diff
@@ -27,6 +28,13 @@ def delete(self, entity_name):
         self.browser.handle_alert()
         view.flash.assert_no_error()
         view.flash.dismiss()
+        view = self.navigate_to(self, 'All')
+        wait_for(
+         lambda: not view.search(entity_name),
```
Comment (oshtaier) [inline airgun/entities/organization.py]: one space indent?

File: airgun/entities/organization.py [hunk 4]
```diff
@@ -27,6 +28,13 @@ def delete(self, entity_name):
         self.browser.handle_alert()
         view.flash.assert_no_error()
         view.flash.dismiss()
+        view = self.navigate_to(self, 'All')
```
Comment (oshtaier) [inline airgun/entities/organization.py]: that part become too common over the code. We need to think about post delete procedure. Maybe create an issue for that

File: airgun/entities/subscription.py [hunk 5]
```diff
@@ -39,6 +39,8 @@ def _wait_for_process_to_finish(self, name, has_manifest=False):
                 handle_exception=True, timeout=10,
                 logger=view.logger
         )
+        # uncomment when BZ#1651981 is fixed
```
Comment (oshtaier) [inline airgun/entities/subscription.py]: what is a reason for that change? Does someone approve that workaround?
If all cases for subscription fail, so bug is a blocker and only after confirmation you can do changes like this

File: airgun/widgets.py (hunk 2 above)
Comment (abalakh) [reply airgun/widgets.py]: No idea how pycharm's search didn't reveal that to me. Big thanks for catching!

File: airgun/entities/organization.py (hunk 4 above)
Comment (abalakh) [reply airgun/entities/organization.py]: Yeah, we already have #19 for that

File: airgun/entities/subscription.py (hunk 5 above)
Comment (abalakh) [reply airgun/entities/subscription.py]: I'm not sure it's a good idea to fail all the tests which are using subscriptions page when all the functionality being tested is working ok.

Comment (oshtaier) [reply airgun/entities/subscription.py]: it is not okay, as user will see 404 error on the screen. We cannot omit that message. For WebUI that  issue is a blocker, so we cannot test functionality if we see anything like this. Except the case when everybody in team is informed and team lead is approve to skip that error

**Comment by abalakh** [comment]
@oshtaier comments addressed.
>pending tests:
>few random for organization

```python
pytest -v tests/foreman/ui_airgun/test_organization.py -k 'test_positive_update_user or test_positive_update_environment or test_positive_update_location'
============================== test session starts ===============================
platform darwin -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.22.0, services-1.2.1, mock-1.10.0, forked-0.2, env-0.6.2
collecting 12 items                                                              2018-11-27 20:32:14 - conftest - DEBUG - BZ deselect is disabled in settings

collected 12 items / 9 deselected

tests/foreman/ui_airgun/test_organization.py::test_positive_update_user PASSED [ 33%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_environment PASSED [ 66%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_location PASSED [100%]

==================== 3 passed, 9 deselected in 253.99 seconds ====================
```

>and for that refresh manifest method.
>If you do them already - just link to them

https://github.com/SatelliteQE/robottelo/pull/6498#issuecomment-442162757

**Pull request merged**